Demo/Controllers/MessageViewController.swift

```diff
@@ -17,14 +17,22 @@ class MessageViewController: UIViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        
+
+        guard let chatkit = self.chatkit else {
+            print("Error: MessageViewController view did load, but Chatkit is not available")
```

Comment:
Printing to console has a significant performance impact for the app (it affects a few journals/logs maintained by the OS). We should avoid that.

Reply:
`fatalError` might be useful here.